snmp_cisco/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+snmp.cefcFRUPowerAdminStatus,gauge,,,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] Administratively desired FRU power state.,0,snmp,
```

Comment:
nice choice, does this also report for ISR, ICM and UC virtual machines?

Reply:
This metric is in the `_generic_cisco` profile, which is used by all of the above families but Meraki - for which I added the other metric.